### PR TITLE
[Kubernetes] more networking details and readable promql

### PR DESCRIPTION
**Pull request by hairmare**

### CODE REVIEW

File: kubernetes-workshop/8004_network/modul_network_8004_01_slides.md [hunk 1]
```diff
@@ -77,9 +90,9 @@ Two types of services allow external access
 ## LoadBalancer
 
 * Requests an external loadbalancer
-* Mostly relevant for cloud environments
+* Mostly relevant for cloud environments and non trivial (ie. production) on-prem deploys
```
Comment (tongpu) [inline kubernetes-workshop/8004_network/modul_network_8004_01_slides.md]: I have the feeling you're missing a for here?

```suggestion
* Mostly relevant for cloud environments and non trivial for (ie. production) on-prem deploys
```

Comment (hairmare) [reply kubernetes-workshop/8004_network/modul_network_8004_01_slides.md]: It's for non trivial on-prem deploys, it's not non trivial.

Comment (tongpu) [reply kubernetes-workshop/8004_network/modul_network_8004_01_slides.md]: Didn't know that trivial could also have that meaning. English is a strange language.

File: kubernetes-workshop/8004_network/modul_network_8004_01_slides.md [hunk 2]
```diff
@@ -55,6 +60,14 @@ Introduction to the networking layers in Kubernetes
 
 ![](static/kubernetes_service_network.png "Kubernetes ClusterIP Network Flow")
 
+##
+
+* Traffic from client process lands on pod veth0
+* Gets routed to a different node, through kube-proxy if needed
```
Comment (tongpu) [inline kubernetes-workshop/8004_network/modul_network_8004_01_slides.md]:
```suggestion
* Get's picked up by iptables chain for that service
* Service ip maps to DNAT for target Pod ip(s)
* Traffic is forwarded to Pod ip
```
Multi-line suggestions don't seem to work that well, but I guess you get the idea. The service chain in iptables is managed by kube-proxy on every node in the cluster.

Comment (hairmare) [reply kubernetes-workshop/8004_network/modul_network_8004_01_slides.md]: > but I guess you get the idea

Not really :wink: 

Do we really want to go to the level of detail in you suggestion?

Comment (tongpu) [reply kubernetes-workshop/8004_network/modul_network_8004_01_slides.md]: We don't have to, but kube-proxy maintains the same iptables configuration on every node, so at least that should be fixed.

**Pull request merged**